dpp/src/tests/utils/error_helpers.rs

```diff
@@ -33,6 +34,23 @@ pub fn get_index_error(consensus_error: &ConsensusError) -> &IndexError {
     }
 }
 
+pub fn get_state_error_from_result(
+    result: &ValidationResult<()>,
+    error_number: usize,
+) -> &StateError {
+    match result
+        .errors
+        .get(error_number)
+        .expect("error should be found")
+    {
+        ConsensusError::StateError(state_error) => state_error,
+        _ => panic!(
```

Comment:
Do we really want that to panic or just return `None`?

Reply:
Meh, maybe change this later.